### ISSUE TITLE
How to override application configuration for this package

### ISSUE THREAD

**Issue by cssriraman** [serilog-contrib/Serilog.Sinks.MicrosoftTeams.Alternative#29]
I have couple of issues as listed below. It would be great if you help me to resolve these issues.

Issue 1:

I have an Asp.net core 6 web api. I host the application in Azure App Services. I have multiple environments like Dev, Test, Staging and Production.

Usually, I will add all properties to appSettings.json which does not change and I specify an empty string or null for properties that may vary across different environments. Then, I specify those properties in the Azure app service configuration.

In our case, only web hook uri will vary across different environment. So, I added the configuration in appSettings.json file as below. But the application failed to start.
```
{
  "Name": "MicrosoftTeams",
  "Args": {
	"webHookUri": "",
	"titleTemplate": "{ApplicationName} - {Environment}",
	"batchSizeLimit": 10,
	"period": "00:00:05",
	"restrictedToMinimumLevel": "Warning",
	"outputTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} {Level:u3} {Message}{NewLine}{Exception}",
	"queueLimit": 10
  }
}
```
I do not want to add bunch of properties to Azure config which does not change. At the same time I do not want to hard code those settings in code.

Let me know if there is a way to fix this issue?

Issue 2:

When the application sends several messages like 50 messages in a min, after a while no messages are sent to teams channel. After some time it starts working again. After googling a lot, found that it may be due to rate limiting with Teams. Is there a way to identify this issue and take any alternative approach can be taken?

Thanks a lot :)

**Comment by SeppPenner** [comment]
I need to check that in detail.

**Comment by SeppPenner** [comment]
Need to check on 1 again, 2 can be resolved by either checking the self log (For debugging) --> There will be a message indicating too many requests `Too many requests`, or you can add the `failureCallback` handler like this (Version 14.3+):

```csharp
failureCallback: e =>
{
    if (e is LoggingException le && le.StatusCode = HttpStatusCode.TooManyRequests)
    {
        // Todo: Do whatever is needed.
    }
}
```

**Comment by SeppPenner** [comment]
Done with https://github.com/serilog-contrib/Serilog.Sinks.MicrosoftTeams.Alternative/commit/952b6272aaab9ea25547e8ddbf27076fd7cd3ad5.

**Comment by SeppPenner** [comment]
Fixed in version 1.4.3 already.
